### PR TITLE
To SDK docs, add locations of binaries

### PR DESCRIPTION
**Pull request by genwhittTTD**
To SDK docs, add locations of binaries. One query for SW on JavaScript SDK.

### CODE REVIEW

File: docs/sdks/uid2-sdk-ref-python.md [hunk 1]
```diff
@@ -42,9 +44,15 @@ This SDK simplifies integration with UID2 for any DSPs or UID2 sharers who are u
 
 The SDK supports Python 3.6 and above.
 
-## SDK Repository
+## GitHub Repository/Binary
```
Comment (sunnywu) [inline docs/sdks/uid2-sdk-ref-python.md:47]: Replace **Binary** with **Package**

File: docs/sdks/uid2-sdk-ref-java.md [hunk 2]
```diff
@@ -41,9 +43,15 @@ This SDK simplifies integration with UID2 for any publishers, DSPs, and UID2 sha
...
 
-## SDK Repository
+## GitHub Repository/Binary
 
-This SDK is available in GitHub: [UID2 SDK for Java](https://github.com/IABTechLab/uid2-client-java/blob/master/README.md).
+This SDK is in the following open-source GitHub repository:
+
+- [UID2 SDK for Java](https://github.com/IABTechLab/uid2-client-java/blob/master/README.md)
+
+The binary is published on Sonatype:
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/sdks/uid2-sdk-ref-java.md:52]: Done, thx.

File: docs/sdks/uid2-sdk-ref-python.md [hunk 3]
```diff
@@ -42,9 +44,15 @@ This SDK simplifies integration with UID2 for any DSPs or UID2 sharers who are u
...
 
-This SDK is available in GitHub: [UID2 SDK for Python](https://github.com/IABTechLab/uid2-client-python/blob/master/README.md).
+This SDK is in the following open-source GitHub repository:
+
+- [UID2 SDK for Python](https://github.com/IABTechLab/uid2-client-python/blob/master/README.md)
+
+The binary is published in this location:
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/sdks/uid2-sdk-ref-python.md:53]: Done. Thx!

File: docs/sdks/uid2-sdk-ref-python.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/sdks/uid2-sdk-ref-python.md:47]: Done. Thx!

**Pull request merged**